packages/app/src/cli/prompts/generate/extension.ts

```diff
@@ -20,82 +20,65 @@ interface GenerateExtensionOutput {
   extensionFlavor?: string
 }
 
-export const extensionFlavorQuestion = (specification: GenericSpecification): ui.Question => {
-  return {
-    type: 'select',
-    name: 'extensionFlavor',
-    message: 'What would you like to work in?',
-    choices: specification.supportedFlavors,
-    default: 'react',
-  }
-}
-
 export function buildChoices(specifications: GenericSpecification[]) {
   return specifications
     .map((type) => {
       const choiceWithoutGroup = {
-        name: type.externalName,
+        label: type.externalName,
         value: type.identifier,
       }
       const group = extensionTypesGroups.find((group) => includes(group.extensions, type.identifier))
       if (group) {
         return {
           ...choiceWithoutGroup,
-          group: {
-            name: group.name,
-            order: extensionTypesGroups.indexOf(group),
```

Comment:
We do lose this order key. Please let me know if it's really important that the order of the groups is preserved.

Reply:
Tophatted it and works as expected.